### PR TITLE
Feature#13 employees crud

### LINKED ISSUE #13: Create controller for Employees
# REQUIREMENTS
We're going to create the controller for the Employees.

# Acceptance Criteria
- [x] Make functional the Create method
- [x] Make functional the Read method
- [x] Make functional the Update method

### PR DESCRIPTION
**Pull request by xFerchoVJ**
# Description
-I've added the resources for the employees in the routes file
- I've create the Employees controller and implement the functions to make the crud.

### CODE REVIEW

File: checador/app/assets/stylesheets/employees.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the employees controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline checador/app/assets/stylesheets/employees.scss]: If you don't occupy it, delete it.

File: checador/app/helpers/employees_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module EmployeesHelper
+end
```
Comment (YilKanda) [inline checador/app/helpers/employees_helper.rb]: If you don't occupy it, delete it.

File: checador/spec/helpers/employees_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the EmployeesHelper. For example:
+#
+# describe EmployeesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (YilKanda) [inline checador/spec/helpers/employees_helper_spec.rb]: If you don't occupy it, delete it.

**Review by YilKanda — COMMENTED** [review]
The code or files that are not necessary in your project, you can delete them.

**Review by jairolan — DISMISSED** [review]

File: checador/app/controllers/employees_controller.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
+class EmployeesController < ApplicationController
+  
+  def index
+    @employees = Employee.all
+  end
+
+  def show
+    set_employee
+  end
+
```
Comment (JoseRicardoJimenez) [inline checador/app/controllers/employees_controller.rb]: You can delete this too. if you don't need it, of course.

**Review by JoseRicardoJimenez — DISMISSED** [review]

File: checador/spec/helpers/employees_helper_spec.rb (hunk 3 above)
Comment (xFerchoVJ) [reply checador/spec/helpers/employees_helper_spec.rb]: Thank you, let me delete it!

File: checador/app/controllers/employees_controller.rb (hunk 4 above)
Comment (xFerchoVJ) [reply checador/app/controllers/employees_controller.rb]: Thank you, let me change it!:)

**Review by JoseRicardoJimenez — APPROVED** [review]
good job (Y)

**Review by SEVERO96 — APPROVED** [review]

**Comment by ManuelAcostaGuerrero** [comment]
good job

**Review by SampabloC — APPROVED** [review]

**Review by SaraNavarro — APPROVED** [review]

**Pull request merged**